internal_ws/ykcompile/src/lib.rs

```diff
@@ -1368,7 +1376,7 @@ impl TraceCompiler {
         // Reserved stack space for spilled locals during execution.
         let soff = self.stack_builder.size();
         // Reserved memory on the stack to spill live locals to during a guard failure.
-        let live_off = REG_POOL.len() * 8;
+        let live_off = REG_POOL.len() * 8; // Is `8` a machine word? Prefer a calculation/constant name.
```

Comment:
It is, and yes that should be a constant.

Reply:
Think this was resolved in 9a21d335171ee9c02cf96bc8938609c43acacfed.